src/jwk.rs

```diff
@@ -116,7 +116,7 @@ impl<'de> Deserialize<'de> for KeyOperations {
         D: Deserializer<'de>,
     {
         struct KeyOperationsVisitor;
-        impl<'de> de::Visitor<'de> for KeyOperationsVisitor {
+        impl de::Visitor<'_> for KeyOperationsVisitor {
```

Comment:
Why lifetime change?

Reply:
https://github.com/Keats/jsonwebtoken/actions/runs/13506827659/job/38042025098?pr=420

Reply:
@jayakasadev, do you know if it's due to the new 1.85.0 elision rules?
I wonder if this change breaks pre-2024 editions.

Reply:
I can't say for sure. I don't have visibility into older runs on `1.84.0` or other `1.85.0` runs. If this came up for the first time, then I would hazard to guess that this is the case.

Reply:
```
In Rust 2024, all in-scope generic parameters, including lifetime parameters, are implicitly captured when the use<..> bound is not present.
```
https://doc.rust-lang.org/edition-guide/rust-2024/rpit-lifetime-capture.html

Reply:
https://medium.com/@TechSavvyScribe/understanding-lifetime-elision-in-rust-5df771c532f7#:~:text=There%20are%20three%20lifetime%20elision%20rules%20that,lifetime%20is%20assigned%20to%20all%20output%20reference

Reply:
The GH actions run the tests against the MSRV here https://github.com/Keats/jsonwebtoken/blob/master/.github/workflows/ci.yml#L40
Can you manually run that workflow in your fork to see if it works?

Reply:
like this?
https://github.com/jayakasadev/jsonwebtoken/pull/1